spec/vrt/map_spec.rb

```diff
@@ -105,5 +105,21 @@
 
       it { is_expected.to be_falsey }
     end
+
+    context 'when ending in a .' do
+      let(:string) { 'server_security_misconfiguration.' }
+
+      it { is_expected.to be_falsey }
+    end
+
+    context 'when a nested node' do
+      let(:string) { 'server_security_misconfiguration.unsafe_cross_origin_resource_sharing' }
```

Comment:
I realized the only reason we prob don't have a fully qualified id here is because we don't want it to be VRT schema dependent. I don't have a solution but this seems very 💩 -y

Reply:
Yup, that's what we did when this code lived in our internal private repo. I'll put up another PR to that effect later